src/protocol/wrapper_generator.cpp

```diff
@@ -391,6 +451,8 @@ class Interface
             emit_indented_lines(out, "    ", {
                 { "uint32_t const max_version;" }
             });
+            if (!methods.empty())
+                out << std::endl;
```

Comment:
We don't need to force a flush(): "out << std::endl;" should be "out << '\n';"

Reply:
Is this case different from the other 60 instances of std::endl? Or should they all eventually be replaced by '\n'?

Reply:
Yes, just a case of not following a bad precedent